packages/main/src/MenuItem.ts

```diff
@@ -158,6 +158,15 @@ class MenuItem extends ListItem {
 	@slot({ "default": true, type: HTMLElement, invalidateOnChildChange: true })
 	items!: Array<MenuItem>;
 
+	/**
+	 * Defines the components that should be displayed at the end of the menu item.
+	 *
+	 * **Note:** It is highly recommended to slot only components of type `ui5-button`,`ui5-link` or `ui5-icon`.
+	 * @public
+	 */
+	@slot({ type: HTMLElement, invalidateOnChildChange: true })
```

Comment:
`invalidateOnChildChange` is redundant. Why did you set it? Every slot is already invalidated if you add new/remove existing children. This setting is only necessary if you want behavior that depends on children changing their properties, f.e. the list must invalidate itself if a list item changes its `mode` property, and is usually necessary for components that render abstract elements. Here I don't see any logic that depends on the state of the children inside the new slot.

Reply:
Ok, will remove it.